src/ws-client.cpp

```diff
@@ -204,6 +208,11 @@ void BaseTarget::initialize(BaseBackend& backend)
     wl_display_roundtrip_queue(display, m_wl.eventQueue);
     wl_registry_destroy(registry);
 
+    if (!m_wl.compositor)
+        g_error("Failed to bind wl_compositor");
+    if (!m_wl.wpe_bridge)
```

Comment:
This should be `wpeBridge`, the compiler complains:

```
../src/ws-client.cpp: In member function ‘void WS::BaseTarget::initialize(WS::BaseBackend&)’:
../src/ws-client.cpp:213:15: error: ‘struct WS::BaseTarget::<unnamed>’ has no member named ‘wpe_bridge’; did you mean ‘wpeBridge’?
  213 |     if (!m_wl.wpe_bridge)
      |               ^~~~~~~~~~
      |               wpeBridge
```

Reply:
(On a related note, the CI somehow did not run for this PR filed from your fork, I have not seen that happen before, but maybe we have something misconfigured.)